### PR TITLE
[interactor] focus action on all elements

### PR DESCRIPTION
**Pull request by jbolda**
This extracts out the focus action and imports it into each interactor (and adds it to a few that were missing it).

### CODE REVIEW

File: packages/interactor/src/focused.ts
Comment (jbolda) [inline packages/interactor/src/focused.ts]: This (and the blur) have an `any` type that I am struggling to narrow. Would appreciate any ideas to address that.

Comment (jnicklas) [reply packages/interactor/src/focused.ts]: I think `HTMLElement` is the appropriate type here instead of `any`. That's what you should use, but I'm not sure if there will be any problems due to subtyping/variance on generics, so TypeScript might reject it.

File: packages/interactor/src/focused.ts
Comment (jnicklas) [inline packages/interactor/src/focused.ts]: Why this complicated generic type? Can we leave this to be inferred?

File: packages/interactor/src/focused.ts
Comment (jbolda) [reply packages/interactor/src/focused.ts]: `<E extends HTMLElement>` was the ticket here. Thanks!

**Comment by cowboyd** [comment]
Interesting that this changed to interactor, triggered a preview of every package /cc @minkimcello

File: packages/interactor/src/focused.ts
Comment (jnicklas) [inline packages/interactor/src/focused.ts]: Since these become public API, maybe we could add some documentation for them?

**Review by jnicklas — APPROVED** [review]
Approved, but would be good to add some docs as well before merging :)

**Comment by minkimcello** [comment]
> Interesting that this changed to interactor, triggered a preview of every package /cc @minkimcello

Hmm I'm not sure why it did that either. I'll keep an eye out to see if it does that for every PR.

**Comment by minkimcello** [comment]
@cowboyd this is a wild guess but I think it might have published all preview packages because we merged a changeset PR recently and this branch might not have rebased yet.

File: packages/interactor/src/focused.ts
Comment (jbolda) [reply packages/interactor/src/focused.ts]: 👍

**Comment by jbolda** [comment]
Added a few doc notes (and did a rebase off of `v0` fyi @minkimcello ).

**Review by cowboyd — APPROVED** [review]

**Pull request merged**
